nym-vpn-app/src/types/app-state.ts

```diff
@@ -32,6 +32,7 @@ export type InitState = {
   welcomeChecked: boolean;
   vpnMode: VpnMode;
   vpnd: VpndStatus;
+  streamingOptimizedLabelSeen: boolean;
```

Comment:
This early init state type, is good for important state bits (only) that need to be prepared as soon as possible and before the React app is spawned.

But that is not the case for `streamingOptimizedLabelSeen`.

To initialize it (using kvGet), you should do it in `initFirstBatch()` https://github.com/nymtech/nym-vpn-client/blob/d4159a0234f76ff18bb5d88bce3e5a2ed85829f7/nym-vpn-app/src/state/init.ts#L40
The advantage of `initFirstBatch()` is that it parallelizes all async calls, including db gets.